models/issues/issue_user.go

```diff
@@ -14,8 +14,8 @@ import (
 // IssueUser represents an issue-user relation.
 type IssueUser struct {
 	ID          int64 `xorm:"pk autoincr"`
-	UID         int64 `xorm:"INDEX"` // User ID.
-	IssueID     int64 `xorm:"INDEX"`
+	UID         int64 `xorm:"INDEX unique(uidtoissue)"` // User ID.
+	IssueID     int64 `xorm:"INDEX unique(uidtoissue)"`
```

Comment:
It would cause inconsistent results.

For old Gitea users, they will have 3 indices in database: `uid`, `issue_id`, `uid, issue_id`.

But for new Gitea users, they will only have 1 index: `uid, issue_id`


BTW: the index name `uidtoissue` really reads strange to me.

Reply:
hm - not sure about the number of indices. I renamed the index - hope thats better now.

According to my test the indices are the same for new and existing installations

I just started a fresh mysql docker container, started gitea and after the install page i had the following indices in my database:

![image](https://github.com/go-gitea/gitea/assets/1135157/cf58c305-8a0d-4a30-af55-44b11252ee36)

After that I created a fresh mysql docker container, started the release binary and installed the database:

![image](https://github.com/go-gitea/gitea/assets/1135157/e29a9041-f09c-42cb-a9f2-b9e79d2e7bba)

and then started the binary with the new migration:

![image](https://github.com/go-gitea/gitea/assets/1135157/12873cac-43cd-40fb-9ec8-52129b5efe0d)

Reply:
Hmm, you are right, I misunderstand the field tag.